### PR TITLE
added remove and last method from index, first and last

### PR DESCRIPTION
**Pull request by pipo411**

### CODE REVIEW

File: src/main/homework1/IList.java [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+
+public interface IList {
+
+    void add(int index, int value);
+
+    void addFirst(int value);
+
+    void addLast(int value);
+
+
```
Comment (jpsandovaln) [inline src/main/homework1/IList.java:32]: remove extra line

File: src/test/homework1/CustomListTest.java [hunk 2]
```diff
@@ -0,0 +1,87 @@
...
+        customList.addFirst(1);
+        customList.add(1, 2);
+        customList.addLast(4);
+        assertEquals(customList.getIndex(0).value, 1);
+        assertEquals(customList.getIndex(1).value, 2);
+        assertEquals(customList.getIndex(2).value, 4);
+
+    }
+
+    private void assertEquals(int value, int i) {
```
Comment (jpsandovaln) [inline src/test/homework1/CustomListTest.java]: ?

File: src/main/homework1/IList.java (hunk 1 above)
Comment (pipo411) [reply src/main/homework1/IList.java:32]: Done

File: src/test/homework1/CustomListTest.java (hunk 2 above)
Comment (pipo411) [reply src/test/homework1/CustomListTest.java]: It was removed, thanks for the observations

**Review by jpsandovaln — APPROVED** [review]

**Pull request merged**